### ISSUE TITLE
error while loading shared libraries

### ISSUE THREAD

**Issue by zone-hai** [lizhuohua/rust-mir-checker#6]
I compiled mir-checker and test it in the tests/safe-bugs/division-by-zero/src/main.rs
but :
> xl@zone:~/rust-mir-checker$ target/debug/mir-checker tests/safe-bugs/division-by-zero/src/main.rs --entry main --domain 
                     interval --widening_delay 5 --narrowing_iteration 5
> target/debug/mir-checker: error while loading shared libraries: librustc_driver-834924640dd700d5.so: cannot open shared 
   object file: No such file or directory

how can i slove it?

**Comment by lizhuohua** [comment]
Hi, try to run this command:
```sh
export LD_LIBRARY_PATH=$(rustc --print sysroot)/lib:$LD_LIBRARY_PATH
```

**Comment by echos2019** [comment]
it cant solve the problem, so I think `cargo install --path .` and `cd tests/safe-bugs/division-by-zero ` and`cargo mir-checker tests/safe-bugs/division-by-zero/src/main.rs --entry main --domain interval --widening_delay 5 --narrowing_iteration 5`can be useful.
but cargo install --path . cant passing

**Comment by ZXXYy** [comment]
same problem, `export LD_LIBRARY_PATH` doesn't work, and `cargo install --path .` fails with the error message 'This version of Cargo is older than the 2021 edition and only supports the 2015 and 2018 editions.'
